plugins/CorePluginsAdmin/Controller.php

```diff
@@ -219,6 +219,16 @@ protected function getPluginsInfo($themesOnly = false)
         foreach ($plugins as $pluginName => &$plugin) {
 
             $plugin['isCorePlugin'] = $this->pluginManager->isPluginBundledWithCore($pluginName);
+            $plugin['isOfficialPlugin'] = false;
+
+            if (isset($plugin['info']) && isset($plugin['info']['authors'])) {
+                foreach ($plugin['info']['authors'] as $author) {
+                    if ($author['name'] == 'Piwik' || $author['name'] == 'InnoCraft') {
```

Comment:
maybe good to check lowercase here as it may change in the future (shouldn't but never know)